project/src/components/app/app.jsx

```diff
@@ -1,7 +1,20 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import HomePage from '../homePage.jsx';
 
-function App() {
-  return <p>Hello, world!</p>;
+function App(props) {
```

Comment:
Давай будем отдавать предпочтение стрелочным функциям
(подправить нужно во всех компонентах)
```suggestion
const App = (props) => {
```